katas/content/superdense_coding/entangled_pair/Verification.qs

```diff
@@ -0,0 +1,16 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Katas;
+
+    operation CreateEntangledPair(qs : Qubit[]) : Unit is Adj + Ctl {
+        H(qs[0]);
+        CNOT(qs[0], qs[1]);
+    }
+
+    operation CheckSolution() : Bool {
```

Comment:
I think this needs to be annotated with `@EntryPoint()`, though I'm not 100% sure whether it's absolutely required. Verification seems to work fine without it in the playground